home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/mapper/IServiceMapperImpl.java

```diff
@@ -0,0 +1,25 @@
+package com.softserveinc.ita.homeproject.homeservice.mapper;
+
+import com.softserveinc.ita.homeproject.homedata.entity.BaseEntity;
+import com.softserveinc.ita.homeproject.homeservice.dto.BaseDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import java.lang.reflect.Type;
+
+@Component
+@RequiredArgsConstructor
+public class IServiceMapperImpl implements IServiceMapper {
```

Comment:
Why do we need it? What if we just use ServiceMapper directly?

Reply:
Again, didn't think about it. Done ;)